AFQ/tractography.py

```diff
@@ -21,7 +21,7 @@
 def track(params_file, directions="prob", max_angle=30., sphere=None,
           seed_mask=None, seed_threshold=0, n_seeds=1, random_seeds=False,
           rng_seed=None, stop_mask=None, stop_threshold=0, step_size=0.5,
-          min_length=10, max_length=1000, odf_model="CSD",
+          min_length=40, max_length=200, odf_model="CSD",
```

Comment:
These are pretty dramatic changes! Can you say a bit more about this?

Reply:
This should bring us back towards mAFQ defaults. I don't remember why we chose 10 and 1000 in the first place, but Jason Maya and I had a conversation about these and in most cases these lengths will serve better. One can adjust min length if they need to. The smaller streamlines were also making cleaning harder by changing the distribution.

Reply:
That makes sense! Thanks for clarifying.